recipe/bld.bat

```diff
@@ -6,8 +6,8 @@ mkdir -p build
 cmake                                                                        ^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"                                ^
     -DCMAKE_BUILD_TYPE=Release                                               ^
-    -DENABLE_INTEL_GNA=OFF                                                   ^
     -DENABLE_SYSTEM_TBB=ON                                                   ^
+    -DENABLE_INTEL_NPU=ON                                                    ^
```

Comment:
it's automatically enabled. We need:
- add new output in conda forge recipe
- add new resources (level zero) which will be used by NPU (similarly to oneDNN for CPU / GPU)